sdk/personalizer/Azure.AI.Personalizer/src/Generated/Models/DateRange.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+
+namespace Azure.AI.Personalizer.Models
+{
+    /// <summary> A date range starting at From and ending at To. </summary>
+    public partial class DateRange
```

Comment:
I believe this functionality is available through [`Azure.Core.Experimental`](https://adpnetsource.azurewebsites.net/#Azure.Core.Experimental/DateTimeRange.cs,754691b454c576d1).

Reply:
Discussed offline, will look into this possibility later.